src/microbe_stage/Microbe.cs

```diff
@@ -1942,14 +2149,41 @@ private Vector3 CalculateNearbyWorldPosition()
         return Translation + (ejectionDirection * ejectionDistance);
     }
 
+    private void MakeChildrenLink(Microbe parent)
```

Comment:
I think these are named a bit badly, because this seems to be about changing the parenting status of this microbe, and not this making links to this object's children, so this name confuses me. I think these could be turned into the methods that MicrobeColony calls on adding and removal of colony members.